elevenlabs_mcp/server.py

```
@@ -1215,6 +1215,75 @@ def create_composition_plan(
 
     return composition_plan
 
+@mcp.tool(description="List all available workspace tools")
+def list_tools(max_length: int = 10000) -> TextContent:
```

Ruff (0.13.2)

Unused function argument: `max_length`

(ARG001)